_docs/Getting-Started.md

```diff
@@ -12,36 +12,34 @@ This article explains how to get started using Coveo Turbo to create, modify, an
 
 > For the full reference of the Coveo Turbo CLI, see [CLI Reference]({{ site.baseurl }}/docs/CLI-Reference).
 
-## Install Coveo Turbo
+## Install Coveo Turbo Prerequisites
 
 To install Coveo Turbo, you need a Node environment with npm. You can download and install both on the [node.js website](https://nodejs.org/en/).
 
 > Coveo Turbo requires at least version 12 of Node.js. While using previous versions may work for certain commands, it's possible that other commands will not work properly.
 
-Once you have installed Node.js, you can run the following command to install and prepare the Coveo Turbo CLI:
-
-```
-npm install --save-dev @coveops/cli
-```
+Once you have installed Node.js, you can create a project and our CLI will take care of all the semantics for you.
```

Comment:
You mean with the command coming after, right?
If that's the case, then just remove this line completely.

Reply:
Yep, ok I'll remove it